Make it possible to configure labels and annotations for PVCs (#1946)
strimzi/strimzi-kafka-operator: documentation/book/appendix_crds.adoc

|https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#toleration-v1-core[Toleration] array
|====

[id='type-PodDisruptionBudgetTemplate-{context}']
### `PodDisruptionBudgetTemplate` schema reference

Used in: xref:type-KafkaBridgeTemplate-{context}[`KafkaBridgeTemplate`], xref:type-KafkaClusterTemplate-{context}[`KafkaClusterTemplate`], xref:type-KafkaConnectTemplate-{context}[`KafkaConnectTemplate`], xref:type-KafkaMirrorMakerTemplate-{context}[`KafkaMirrorMakerTemplate`], xref:type-ZookeeperClusterTemplate-{context}[`ZookeeperClusterTemplate`]

[options="header"]
|====
|Property               |Description

> |metadata        1.2+<.<|Metadata which should be applied to the `PodDistruptionBugetTemplate` resource.

Reviewer: ```suggestion
|metadata        1.2+<.<|Metadata to apply to the `PodDistruptionBugetTemplate` resource.
```

|xref:type-MetadataTemplate-{context}[`MetadataTemplate`]

> |maxUnavailable  1.2+<.<|Maximum number of unavailable pods to allow voluntary Pod eviction. A Pod eviction will only be allowed when "maxUnavailable" or fewer pods are unavailable after the eviction. Setting this value to 0 will prevent all voluntary evictions and the pods will need to be evicted manually. Defaults to 1.

Reviewer: ```suggestion
|maxUnavailable  1.2+<.<|Maximum number of unavailable pods to allow voluntary Pod eviction. A Pod eviction is allowed when the `maxUnavailable` number of pods or fewer are unavailable after the eviction. Setting this value to 0 prevents all voluntary evictions, so the pods must be evicted manually. Defaults to 1.
```

> |maxUnavailable  1.2+<.<|Maximum number of unavailable pods to allow voluntary Pod eviction. A Pod eviction will only be allowed when "maxUnavailable" or fewer pods are unavailable after the eviction. Setting this value to 0 will prevent all voluntary evictions and the pods will need to be evicted manually. Defaults to 1.

Reviewer: I didn't quite get the voluntary opposed to manual here: `Setting this value to 0 prevents all voluntary evictions, so the pods must be evicted manually. Defaults to 1.`

If I substitute voluntary for automatic, it makes more sense.
